R/editing.R

```diff
@@ -715,6 +715,11 @@ set_file_name <- function(mn, pid, name) {
             nchar(name) > 0)
 
   sysmeta <- dataone::getSystemMetadata(mn, pid)
+  if (!is.na(sysmeta@fileName)) {
```

Comment:
Could you please add a line of whitespace above the `if` block to keep the code style consistent?